modules/account-lib/test/unit/coin/stx/transaction.ts

```diff
@@ -0,0 +1,49 @@
+import should from 'should';
+import { coins } from '@bitgo/statics';
+import { Transaction } from '../../../../src/coin/stx/transaction';
+import * as testData from '../../../resources/stx/stx';
+import { KeyPair } from '../../../../src/coin/stx/keyPair';
+
+describe('Stx Transaction', () => {
+  const coin = coins.get('stx');
+
+  function getTransaction(): Transaction {
+    return new Transaction(coin);
+  }
+
+  it('should throw empty transaction', () => {

```

Comment:
It would be better to test that a specific error type/message is thrown, instead of any error.